helpdesk-maker-checker-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/makerchecker/model/check/Checker.java

```diff
@@ -25,7 +26,7 @@
    */
   Set<SymMessage> buildSymCheckerMessages(SymMessage symMessage, Object opaque);
 
-  Set<SymMessage> makeApprovedMessages(MakerCheckerMessage makerCheckerMessage, SymMessage symMessage);
+  SymAttachmentInfo getApprovedAttachment(MakerCheckerMessage makerCheckerMessage, SymMessage symMessage);
```

Comment:
You should keep the old interface. The checker should be responsible for building approved messages.